src/axom/quest/SignedDistance.hpp

```diff
@@ -52,7 +53,18 @@ struct UcdMeshData
   /*!
    * \brief Returns the type of the cell at the given index.
    */
-  AXOM_HOST_DEVICE mint::CellType getCellType(IndexType cellId) const;
+  AXOM_HOST_DEVICE mint::CellType getCellType(IndexType cellId) const {
```

Comment:
Would this function need to be marked as `inline` ?

Reply:
I believe member functions defined inside a class scope are implicitly `inline` already

Reply:
Thanks, I missed that it was a member function (I thought it was a standalone).